src/Components/Searcch/Search.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { searchCurrency } from '../../Redux/CurrencySlice';
+import './Search.css';
+
+function Search() {
+  const dispatch = useDispatch();
+  const handleSearch = (e) => {
+    const searchValue = e.target.value.toLowerCase();
+    dispatch(searchCurrency(searchValue));
+  };
+  return (
+    <input
+      type="search"
+      onChange={handleSearch}
+      placeholder="Enter Country"
```

Comment:
- [ ] We are dealing with currencies here, not countries. I think it would be better to use something like `Enter Currency` so as not to confuse users.